common/test/acceptance/tests/video/test_video_events.py

```diff
@@ -47,33 +94,193 @@ def is_video_event(event):
+            'session'
+        )
+        for field in dynamic_string_fields:
+            self.assert_field_type(load_video_event, field, basestring)
+            self.assertIn(field, load_video_event, '{0} not found in the root of the event'.format(field))
+            del load_video_event[field]
+
+        # A weak assertion for the timestamp as well
+        self.assert_field_type(load_video_event, 'time', datetime.datetime)
+        del load_video_event['time']
+
+        # Note that all unpredictable fields have been deleted from the event at this point
+
+        course_key = CourseKey.from_string(self.course_id)
+        static_fields_pattern = {
+            'context': {
+                'course_id': unicode(course_key),
+                'org_id': course_key.org,
+                'path': '/event',
+                'user_id': self.user_info['user_id']
+            },
+            'event_source': 'browser',
+            'event_type': 'load_video',
+            'username': self.user_info['username'],
+            'page': self.browser.current_url,
+            'referer': self.browser.current_url,
+            'name': 'load_video',
+        }
+        assert_events_equal(static_fields_pattern, load_video_event)
+
+
+@ddt.ddt
+class VideoBumperEventsTest(VideoEventsTestMixin):
+    """ Test bumper video event emission """
+
+    # helper methods
+    def seek_video_and_skip(self):
+        """
+        Wait 5 seconds and press "skip" button.
+        """
+        self.video.wait_for_position('0:05')
+        self.video.click_player_button('skip_bumper')
+
+    def seek_video_and_dismiss(self):
+        """
+        Wait 5 seconds and press "do not show again" button.
+        """
+        self.video.wait_for_position('0:05')
+        self.video.click_player_button('do_not_show_again')
+
+    def wait_for_state(self, state='finished'):
+        """
+        Wait until video will be in given state.
+
+        Finished state means that video is played to the end.
+        """
+        self.video.wait_for_state(state)
+
+    def add_bumper(self):
+        """
+        Add video bumper to the course.
+        """
+        additional_data = {
+            u'video_bumper': {
+                u'value': {
+                    "transcripts": {},
+                    "video_id": "edx_video_id"
+                }
+            }
+        }
+        self.course_fixture.add_advanced_settings(additional_data)
+
+    @ddt.data(
+        ('edx.video.bumper.skipped', seek_video_and_skip),
+        ('edx.video.bumper.dismissed', seek_video_and_dismiss),
+        ('edx.video.bumper.stopped', wait_for_state)
+    )
+    @ddt.unpack
+    def test_video_control_events(self, event_type, action):
+        """
+        Scenario: Video component is rendered in the LMS in Youtube mode without HTML5 sources
+        Given the course has a Video component in "Youtube" mode
+        And I play the video
+        And I watch 5 seconds of it
+        And I pause the video
```

Comment:
The scenario here doesn't match the behavior.   The watch-and-pause is now incorporated into the action, and the events emitted depend on the event_type passed in.